packages/flutter/lib/src/material/banner.dart

```diff
@@ -42,7 +45,7 @@ class MaterialBanner extends StatelessWidget {
     this.leadingPadding,
     this.forceActionsBelow = false,
   }) : assert(content != null),
-       assert(actions != null),
+       assert(actions != null && actions.isNotEmpty),
```

Comment:
Sadly, a const constructor is preferable to checking this here. The isNotEmpty assert can go at the the top of the build method.

Reply:
gotcha, done